webviews/components/sidebar.tsx

```diff
@@ -59,50 +60,55 @@ export default function Sidebar({ reviewers, labels, hasWritePermission, isIssue
 					<div className="section-title">Assignees</div>
 					{hasWritePermission ? (
 						<button
+							className="icon-button"
 							title="Add Assignees"
 							onClick={async () => {
 								const newAssignees = await addAssignees();
 								updatePR({ assignees: pr.assignees.concat(newAssignees.added) });
 							}}
 						>
-							{plusIcon}
+							{settingsIcon}
 						</button>
 					) : null}
 				</div>
 				{assignees && assignees.length ? (
 					assignees.map((x, i) => {
 						return (
 							<div key={i} className="section-item reviewer">
-								<Avatar for={x} />
-								<AuthorLink for={x} />
+								<div className="avatar-with-author">
+									<Avatar for={x} />
+									<AuthorLink for={x} />
+								</div>
 								{hasWritePermission ? (
-									<>
-										{nbsp}
-										<button
-											className="push-right remove-item"
-											onClick={async () => {
-												await removeAssignee(x.login);
-											}}
-										>
-											{deleteIcon}️
-										</button>
-										{nbsp}
-									</>
+									<button
+										className="icon-button"
+										onClick={async () => {
+											await removeAssignee(x.login);
+										}}
+									>
+										{deleteIcon}️
+									</button>
 								) : null}
 							</div>
 						);
 					})
 				) : (
 					<div className="section-placeholder">
-						None yet{pr.canEdit ? (
+						None yet
+						{pr.canEdit ? (
```

Comment:
```suggestion
						{hasWritePermission ? (
```

This bug was there before, but the fact that you can edit a PR doesn't mean that you can assign yourself to it.

Reply:
I think this is best handled in a scoped PR. I'd prefer not to start making functional changes in this PR.